pivot/templates/major-gpa.html

```diff
@@ -10,48 +10,52 @@ <h1 class="page-title">View Major GPAs</h1>
-						        <!-- JS generated college list goes in ul -->
-						        <ul class="dropdown-menu" aria-labelledby="dropdownMenu">
-						        </ul>
-						      </div><!-- .input-group-btn -->
-							  <!-- major search -->
-
-						     <div class="form-group">
-							      <label for="search" class="sr-only" >Enter a major name to look up</label>
-								  <input id="search" type="text" class="form-control" name="searchQuery" value="" size="" tabindex="0" data-save-value="true" data-role="Control" data-control_for="search" placeholder="e.g. Biology, English, etc." autocomplete="off">
-								  <span role="status" aria-live="polite" class="ui-helper-hidden-accessible"> </span>
-
-						      <!-- search results go in #suggestions -->
-					            <div id="suggestions"></div>
-					            <!-- hidden div to track selected majors -->
-                                <div class="selected"></div>
-							</div>	<!-- .form-group for Search label +input -->
-						      <div class="input-group-btn">
-							      	<button class="btn btn-default btn-search" type="submit" id="goBtn"><i class="glyphicon glyphicon-search" aria-hidden="true"></i><span class="sr-only">search</span></button>
-							  </div><!--Go button .input-group-btn -->
-
-                          	</div><!--  .input-group -->
-                          	<div class="clear-btn"><button type="button" class="btn btn-link" id="clear_majors" label="clear selected majors" aria-label='clear selected majors'>Clear selections</button>
-                          	</div>
-                          </div><!--  .col-xs-12 col-sm-10??? -->
-                        </div><!-- end of .row -->
-					  </div><!-- of .container -->
+							<div class="col-xs-12 col-sm-2"><!-- TODO ADD BOOTSTRAP CLASS TO REMOVE LEFT & RIGHT GUTTERS -->
+							    <div class="form-group">
+							        <!-- college search: button and ul tags create dropdown -->
+							        <div class="btn-group" id="college-dropdown" style="width:100%">
+							            <button class="btn btn-default" type="button" id="dropdownMenu" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true" style="width:100%;text-align:left;">
+							            All
+							            <span class="caret"></span>
+							            </button>
+							            <!-- JS generated college list goes in ul -->
+							            <ul class="dropdown-menu" aria-labelledby="dropdownMenu">
+							            </ul>
+							        </div>
+							    </div>
+							</div>
+							<div class="col-xs-12 col-sm-5"><!-- TODO ADD BOOTSTRAP CLASS TO REMOVE LEFT & RIGHT GUTTERS -->
+							    <div class="form-group">
+							        <!-- major search -->
+							        <label class="sr-only" for="search">Major</label>
+									<div class="sr-only" id="search-description">Enter the name of a major. Matches results will be shown below as you type.</div>
```

Comment:
why not using the <label>?

Reply:
I don't see a "?" anywhere on the screen. What "?" are you talking about?

Reply:
I am using the `<label>`, see above. 

The search-description just adds extra information for screen readers and is referenced by the search input's attribute "describedby". When the user activates the search it should be read and provide more information about how to use it or what it is.